docs/rst/notes/notes.rst

```diff
@@ -7,24 +7,20 @@
 .. .. include:: forthcoming_version.rst
 
 ##############
-Version v1.1.0
+Version v1.2.0
 ##############
 
-This release includes the following **features**:
-
-* New :ref:`user_manual_user_interface_log_verbosity_argument` and :ref:`user_manual_user_interface_log_filter_argument`
-  arguments to configure *DDS Router* and *Fast DDS* logs shown in application.
-
 This release includes the following **CI improvements**:
 
-* Add *Address Sanitizer* check to all tests.
-* Add *Thread Sanitizer* check to all tests.
-* Compile with `-Wall` flag all CI jobs.
+* Add blackbox tests for repeater in WAN case.
+* Add unittest tests for ``PayloadPool``, ``FastPayloadPool`` and ``MapPayloadPool`` clases.
```

Comment:
Are these tests in this release?

Reply:
They were added in #336 but the tests for MapPayloadPool and FastPayloadPool were just edited so I decided to remove them.